### PR TITLE
Added Notebooks from SNAP

### PR DESCRIPTION
**Pull request by joni-herttuainen**
Fixed the notebooks, checked that they passed.

Changed the checking if resources are synced to be totally independent of `nexusforge` as the checking the value `kgforge.Resource._synchronized` did not work as expected anymore.

### CODE REVIEW

File: notebooks/01_nexus_setup.ipynb [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+     ]
+    }
+   ],
+   "source": [
+    "TOKEN = getpass.getpass()"
```
Comment (eleftherioszisis) [inline notebooks/01_nexus_setup.ipynb:71]: Coul bluepyentity's get_token be used here?

File: notebooks/02_nexus_searching.ipynb [hunk 2]
```diff
@@ -0,0 +1,355 @@
...
+     ]
+    }
+   ],
+   "source": [
+    "import getpass\n",
+    "from bluepyentity.nexus import NexusHelper\n",
+    "\n",
+    "BUCKET = \"nse/test\"\n",
+    "TOKEN = getpass.getpass()\n",
```
Comment (eleftherioszisis) [inline notebooks/02_nexus_searching.ipynb:32]: Same here

File: notebooks/03_nexus_querying.ipynb [hunk 3]
```diff
@@ -0,0 +1,307 @@
...
+     ]
+    }
+   ],
+   "source": [
+    "import getpass\n",
+    "from bluepyentity.nexus import NexusHelper\n",
+    "\n",
+    "BUCKET = \"nse/test\"\n",
+    "TOKEN = getpass.getpass()\n",
```
Comment (eleftherioszisis) [inline notebooks/03_nexus_querying.ipynb:34]: Same here

File: notebooks/04_nexus_displaying_data.ipynb [hunk 4]
```diff
@@ -0,0 +1,1170 @@
...
+     ]
+    }
+   ],
+   "source": [
+    "import getpass\n",
+    "from pprint import pprint\n",
+    "from bluepyentity.nexus import NexusHelper\n",
+    "\n",
+    "BUCKET = \"nse/test\"\n",
+    "TOKEN = getpass.getpass()\n",
```
Comment (eleftherioszisis) [inline notebooks/04_nexus_displaying_data.ipynb:35]: Same here

File: notebooks/05_nexus_browsing_data.ipynb [hunk 5]
```diff
@@ -0,0 +1,445 @@
...
+     ]
+    }
+   ],
+   "source": [
+    "import getpass\n",
+    "from bluepyentity.nexus import NexusHelper\n",
+    "\n",
+    "BUCKET = \"nse/test\"\n",
+    "TOKEN = getpass.getpass()\n",
```
Comment (eleftherioszisis) [inline notebooks/05_nexus_browsing_data.ipynb:34]: Same here

File: notebooks/06_nexus_utilizing_data.ipynb [hunk 6]
```diff
@@ -0,0 +1,964 @@
...
+     ]
+    }
+   ],
+   "source": [
+    "import getpass\n",
+    "from pathlib import Path\n",
+    "from bluepyentity.nexus import NexusHelper\n",
+    "\n",
+    "BUCKET = \"public/hippocampus-hub\"\n",
+    "TOKEN = getpass.getpass()\n",
```
Comment (eleftherioszisis) [inline notebooks/06_nexus_utilizing_data.ipynb:35]: Same here

File: notebooks/in_progress/07_nexus_bluepyemodel.ipynb [hunk 7]
```diff
@@ -0,0 +1,206 @@
...
+     ]
+    }
+   ],
+   "source": [
+    "import getpass\n",
+    "from pathlib import Path\n",
+    "from bluepyentity.nexus import NexusHelper\n",
+    "\n",
+    "BUCKET = \"demo/emodel_pipeline\"\n",
+    "TOKEN = getpass.getpass()\n",
```
Comment (eleftherioszisis) [inline notebooks/in_progress/07_nexus_bluepyemodel.ipynb:35]: Same here

File: notebooks/01_nexus_setup.ipynb (hunk 1 above)
Comment (joni-herttuainen) [reply notebooks/01_nexus_setup.ipynb:71]: Yes, very likely. I was considering trying this but decided that I'd just bring these over and check they still work. 

I'll do any changes/improvements in another PR. Created an issue of this: #20

Comment (eleftherioszisis) [reply notebooks/01_nexus_setup.ipynb:71]: Sure, fair enough

**Review by eleftherioszisis — APPROVED** [review]

**Pull request merged**